ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -752,6 +811,24 @@ export class PrepareMigrationComponent implements OnInit {
     if (this.selectedMigrationType === MigrationTypes.lowDowntimeMigration) {
       this.isLowDtMigrationRunning = true
     }
+    if (this.selectedMigrationType === MigrationTypes.dataprocMigration) {
+      this.isDprocMigrationRunning = true
+    }
+  }
+
+  fetchDataprocJobs() {
+    this.fetch.getDataprocJobs().subscribe({
+      next: (dprocJobs: IDataprocJobs) => {
+        this.isDataprocJobsGenerated = true
+        this.dataprocJobsGenerated = dprocJobs
+      },
+      error: (err: any) => {
+        this.snack.openSnackBar(err.error, 'Close')
+      },
+    })
+
+    this.isDprocMigrationRunning = true
+    
   }
 
   markMigrationComplete() {
```

Comment:
where are we marking migration complete for dataproc migration without any foreign keys?

Reply:
Tested: Migration completion progress showing up in UI for non foreign key migration.
`subscribeMigrationProgress()` >> `if (res.ProgressStatus == ProgressStatus.DataMigrationComplete)`